camelConnector/src/test/java/io/vantiq/extsrc/camelconn/discover/VantiqComponentResolverTest.java

```diff
@@ -731,6 +754,8 @@ public static TriFunction<CamelContext, String, Object, Boolean> defineVerifyOpe
                         if (rawData instanceof Map){
                             //noinspection unchecked,rawtypes
                             msgMap = (Map) rawData;
```

Comment:
little bit of cleanup as we were getting unexpected errors due to unrelated change